tests/baselines/reference/importCallExpressionGrammarError.js

```diff
@@ -7,13 +7,11 @@ import(...["PathModule"]);
 
 var p1 = import(...a);
 const p2 = import();
-const p3 = import(,);
```

Comment:
Had to remove this line as it's now a *parse* error and not a grammar error, and a parse error would make all of the grammar errors disappear (see `grammarErrorAtPos` in `checker.ts`).